_pages/en/strategy-build-vs-buy-explained.md

```diff
@@ -17,406 +17,79 @@ permalink: /strategy-build-vs-buy-explained.html
-    - [Use open standards and solutions](#use-open-standards-and-solutions)
-    - [Work in the open by default](#work-in-the-open-by-default)
-    - [Collaborate widely](#collaborate-widely)
-    - [Empower staff to deliver better services](#empower-staff-to-deliver-better-services)
-  - [Digital Operations Strategic Plan](#digital-operations-strategic-plan)
-    - [A service-oriented government with a user-centred](#a-service-oriented-government-with-a-user-centred)
-    - [An open, collaborative and accessible government](#an-open-collaborative-and-accessible-government)
-    - [Modern technology and modern information practices](#modern-technology-and-modern-information-practices)
-    - [Good digital governance](#good-digital-governance)
-  - [Alignment with D9 Charter](#alignment-with-d9-charter)
-    - [User needs](#user-needs-1)
-    - [Open standards](#open-standards-1)
-    - [Open source](#open-source-1)
-    - [Open markets](#open-markets-1)
-    - [Open government](#open-government-1)
-  - [Policy on Service and Digital](#policy-on-service-and-digital-1)
-  - [Directive on Service and Digital - Appendix A: Mandatory Procedures for Enterprise Architecture Assessment](#directive-on-service-and-digital---appendix-a-mandatory-procedures-for-enterprise-architecture-assessment-1)
-    - [Business Architecture: Design for users first and deliver with multidisciplinary teams](#business-architecture-design-for-users-first-and-deliver-with-multidisciplinary-teams)
-    - [Application Architecture: Use Open Standards and Solutions by Default](#application-architecture-use-open-standards-and-solutions-by-default-1)
-    - [Application Architecture: Enable Interoperability](#application-architecture-enable-interoperability-1)
-  - [Government of Canada Architectural Standards](#government-of-canada-architectural-standards)
-    - [Enable Interoperability](#enable-interoperability)
-- [References](#references)
+- [Introduction](#introduction)
+  - [Scope of This Document](#scope-of-this-document)
+  - [Use Cases](#use-cases)
+    - [Web browser](#web-browser)
+    - [SaaS](#saas)
+    - [Multi Benefit Delivery Management System](#multi-benefit-delivery-management-system)
 
 ## Context
 
 This document is complementary content for the [Build vs Buy Strategy](strategy-build-vs-buy.html).
 
-### Historical Context<!-- omit in toc -->
+## Introduction
 
-The Canadian government's rationale for procuring 3rd party software solutions and support has historically been based on reducing risk and cost for taxpayers.
+Often when we talk about policy we use complex language in order to effectively encompass all ideas and information to the user. As a result however, this makes policy difficult to digest and comprehend. The goal of the IITB Build vs Buy strategy is to provide a new way of thinking about IT procurement and risk management. As such, the following scenarios were created in order to help the user put the framework into context.
```

Comment:
Suggested change to 'As a result however....' sentence.... 'However this can result in policy that is difficult to digest and comprehend'.

Reply:
We should remove "we" from the document, as it is a policy/strategy and not the teams opinion, or a discussion